examples/evaluate_model.py

```diff
@@ -135,14 +136,9 @@ def export_model_preds(preds_file, predictions, validation_stream, verbalizer):
 if __name__ == "__main__":
     configure_random_seed_and_logging()
     args = parse_args()
-    if not all(metric_name in SUPPORTED_METRICS for metric_name in args.metrics):
-        raise KeyError(
-            "One or more requested metrics {} not supported! Supported metrics: {}".format(
-                args.metrics, list(SUPPORTED_METRICS.keys())
-            )
-        )
-    metric_funcs = list(SUPPORTED_METRICS.values())
+    metric_funcs = [SUPPORTED_METRICS[metric] for metric in args.metrics]
```

Comment:
nit: Lets add the `keyerror` validation back, in case there is a typo or something when kicking off evaluation.

Reply:
My idea here was for it to checked by argparse via 'options' so if you make a typo it will fail at the parseargs call. Do you want an additional level of checking?

Reply:
Oh yes, that make sense. Sounds good..